src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -634,7 +634,8 @@ private function getWriteAccessInfo(string $class, string $property, $value): ar
             if ($this->isMethodAccessible($reflClass, $setter, 1)) {
                 $access[self::ACCESS_TYPE] = self::ACCESS_TYPE_METHOD;
                 $access[self::ACCESS_NAME] = $setter;
-            } elseif ($this->isMethodAccessible($reflClass, $getsetter, 1)) {
+            } elseif ($this->isMethodAccessible($reflClass, $getsetter, 1)
+                      || $this->isMethodAccessible($reflClass, $getsetter, 0)) {
```

Comment:
no need for this second call: turning the `1` to a `0` on the existing call

Reply:
Setter-getters are supposed to be supported, because the error message that got me here was:

```
Could not determine access type for property "nationalId" in class
"TimeTracking\Model\WorkerIdentification": Neither the property "nationalId"
nor one of the methods "addNationalId()"/"removeNationalId()",
"setNationalId()", "nationalId()", "__set()" or "__call()" exist and have
public access in class "TimeTracking\Model\WorkerIdentification"..      
```

I don't think it's a new feature. Look at the jQuery comment in the code. It was intended to work like this. Regardless, my project is on 4.1. I'll be glad enough if this gets merged. 

Is this commit really breaking all those tests?

What happens if i fix the style issues in the same branch in my fork?
Do I need to update the pull-request somehow?

Reply:
> I don't think it's a new feature. Look at the jQuery comment in the code. It was intended to work like this. Regardless, my project is on 4.1. I'll be glad enough if this gets merged.

The comment only implies that the method can be called without an argument to work as a getter. But this must be achieved by making the argument (which is used to make the method work as a setter) optional. Being able to completely use a signature without any argument is indeed a new feature.